parcels/examples/documentation_indexing.ipynb

```diff
@@ -184,33 +202,20 @@
+    "\n",
+    "To load B-grid data, you can use the model-specific convenience method `FieldSet.from_b_grid_dataset`."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Background info: How does Parcels deal with B and C grids?\n",
-    "\n",
-    "To keep it simple, let's take the case of a 2D Arakawa-C grid.\n",
-    "\n",
-    "Why doesn't Parcels need the exact location of the `U` and `V` nodes? This is because in C grids, `U` and `V` nodes are always located the same way: for each grid cell, U nodes are located at the middle of the left and right edges, and V nodes are located at the bottom and top edges (see right panel in the Figure below). \n",
-    "\n",
-    "![Arakawa Grid layouts](http://oceanparcels.org/images/DelandmeterVanSebilleFig2.png)\n",
-    "\n",
-    "The only information that Parcels needs is the location of the 4 corners of the cell, which are called the f-nodes. Those are the ones provided by `U.grid` (and are equal to the ones in `V.grid`).\n",
-    "\n",
-    "Now, as you've noticed on the grid illustrated on the figure, there are 4x4 cells. The grid providing the cell corners is a 5x5 grid.\n",
-    "But there are only 4x5 U nodes and 5x4 V nodes, since the grids are staggered.\n",
-    "\n",
-    "The indexing notation chosen in Parcels is the same as for Nemo, `U[j, i]` is located between the cell corners `[j-1, i]` and `[j, i]`, and  `V[j, i]` is located between cell corners `[j, i-1]` and `[j, i]`. This implies that the first row of `U` data and the first column of `V` data is never used (and do not physically exist), but the `U` and `V` fields are correctly provided on a 5x5 table. If your original data are provided on a 4x5 `U` grid and a 5x4 `V` grid, you need to regrid your table to follow Parcels notation before creating a FieldSet!\n",
-    "\n",
-    "For 3D C grids and B grids, the idea is the same. It is important to follow the indexing notation, which is defined in Parcels and in [Delandmeter and Van Sebille (2019)](https://www.geosci-model-dev.net/12/3571/2019/gmd-12-3571-2019.html). \n",
+    "### 3D C- and B-grids\n",
     "\n",
-    "Note that Parcels can take care of loading these B and C grid data for you, through the general `FieldSet.from_c_grid_dataset` and `FieldSet.from_b_grid_dataset` methods, and the model-specific methods `FieldSet.from_nemo` and `FieldSet.from_pop`."
+    "For 3D C-grids and B-grids, the idea is the same. It is important to follow the indexing notation, which is defined in Parcels and in [Delandmeter and Van Sebille (2019)](https://www.geosci-model-dev.net/12/3571/2019/gmd-12-3571-2019.html). In 3D C-grids, the vertical (`W`) velocities at the top and bottom.  Note that in the vertical, the bottom velocity is often omitted, since a no-flux boundary conditions implies a zero vertical velocity at the ocean floor. That means that the vertical dimension of `W` often corresponds to the amount of vertical levels."
```

Comment:
```suggestion
    "For 3D C-grids and B-grids, the idea is the same. It is important to follow the indexing notation, which is defined in Parcels and in [Delandmeter and Van Sebille (2019)](https://www.geosci-model-dev.net/12/3571/2019/gmd-12-3571-2019.html). In 3D C-grids, the vertical (`W`) velocities are at the top and bottom.  Note that in the vertical, the bottom velocity is often omitted, since a no-flux boundary conditions implies a zero vertical velocity at the ocean floor. That means that the vertical dimension of `W` often corresponds to the amount of vertical levels."
```